examples/rmg/catalysis/ch4_o2/simulate_ch4o2cat_RMS.ipynb

```diff
@@ -0,0 +1,508 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b1f80c5e",
+   "metadata": {},
+   "source": [
+    "# Import packages\n",
+    "Note: This notebook can take a while (~5 min) for the first time due to compilation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "82bfbe9d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pyrms import rms\n",
+    "from diffeqpy import de\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from julia import Main\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c1fa0586",
+   "metadata": {},
+   "source": [
+    "Note: You will learn more about how to RMG-cat and generate catalyic mechanism in tomorrow's lecture \"RMG-Cat Systems and Surface Reactions\". You can also remove the `#` in the following cell and run it. This can take quite a while. This is an example for docker users."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "f9caf172",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ! python-jl /rmg/RMG-Py/rmg.py /rmg/RMG-Py/examples/rmg/catalysis/ch4_o2/input.py"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c3cc81ab",
+   "metadata": {},
+   "source": [
+    "## Load the mechanism \n",
+    "<p style=\"color: red;\"> [User input required] </p>\n",
+    "\n",
+    "Load the mechanism file from a `.rms` file\n",
+    "```\n",
+    "phase_dict = rms.readinput(\"/rmg/RMG-Py/examples/rmg/catalysis/ch4_o2/chem41.rms\")\n",
+    "```\n",
+    "where `mechanism_path` is the path to the generated rms input file\n",
+    "\n",
+    "P.S. The same function also works for chemkin files (with thermochemistry and kinetics in the same file):\n",
+    "\n",
+    "```\n",
+    "phase_dict = rms.readinput(\"/rmg/RMG-Py/examples/rmg/catalysis/ch4_o2/chemkin/chem_annotated.inp\"; \n",
+    "                           spcdict=\"/rmg/RMG-Py/examples/rmg/catalysis/ch4_o2/chemkin/species_dictionary.txt\")\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "id": "76557080",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "12 Gas-phase species\n",
+      "3 Gas-phase reactions\n",
+      "29 Surface species\n",
+      "198 Surface reactions\n",
+      "21 Gas-Surface reactions\n"
+     ]
+    }
+   ],
+   "source": [
+    "mechanism_path = \"ch4o2cat.rms\"  # You may change the path to actual mechanism file path\n",
+    "\n",
+    "phase_dict = rms.readinput(mechanism_path)\n",
+    "gas_spcs = phase_dict[\"gas\"][\"Species\"];\n",
+    "gas_rxns = phase_dict[\"gas\"][\"Reactions\"];\n",
+    "surface_spcs = phase_dict[\"surface\"][\"Species\"]\n",
+    "surface_rxns = phase_dict[\"surface\"][\"Reactions\"]\n",
+    "interface_rxns = Main.get(phase_dict, Main.Set([\"surface\", \"gas\"]), Main.nothing)[\"Reactions\"];\n",
```

Comment:
Do you mind adding a few more comments about loading mechanisms? At least, I found the `interface_rxns ... ` line is less readable.